pkg/cli-runtime/testing/command_table.go

```diff
@@ -82,6 +91,9 @@ type CommandTestCase struct {
 	// GivenObjects represents resources that would already exist within Kubernetes. These
 	// resources are passed directly to the fake client.
 	GivenObjects []client.Object
+	// UnstructuredObjects represents resources that would already exist within Kubernetes. These
+	// resources are passed directly to the fake client.
+	UnstructuredObjects []client.Object
```

Comment:
you do not need this anymore. You are checking givenObject kind to decide whether to use builder

Reply:
I have removed the `unstructuredObjects`